cdm/src/main/java/ucar/nc2/ft2/simpgeometry/CFLine.java

```diff
@@ -22,6 +26,7 @@
  */
 public class CFLine implements Line {
 
+	private static final Logger cfl = LoggerFactory.getLogger(CFLine.class);

```

Comment:
Nice SLF4J usage!

In many projects there's a convention of naming `Logger` instances something like `log`/`LOG`/ `logger`/`LOGGER` to improve clarity of intent. When `Logger` instances have the same name across classes, it becomes easy to read calls to log methods and to add new calls to log methods without looking up what each class names their `Logger` instance.

It is important to highlight that this naming convention is particular to loggers.  The convention of giving the same name to all instances of a type across classes does not necessarily generalize to other use cases. In fact it often makes sense to give variables of the same type a different name in different classes depending on context. However, consistent naming of Loggers is often appealing because they are a ubiquitous, consistent piece of infrastructure peppered throughout codebases.